e2e-polybft/cardanofw/test-cardano-chain.go

```diff
@@ -266,30 +270,21 @@ func (ec *TestCardanoChain) FundWallets(ctx context.Context) error {
 		fmt.Printf("%s fee addr funded: %s\n", GetNetworkName(ec.config), txHash)
 	}
 
-	if ec.config.InitialHotWalletTokenAmount != nil && ec.config.InitialHotWalletTokenAmount.Uint64() != 0 {
+	if ec.config.FundTokenAmount != 0 || ec.config.FundAmount != 0 {
```

Comment:
Why don't we have separate funds (transactions) for tokens and lovelace? I mean, one UTxO exclusively for tokens and one for lovelace? If this is better for testing, let's leave it as is